rocketpy/utilities.py

```diff
@@ -654,3 +655,25 @@ def liftoff_speed(mass):
     if plot:
         retfunc.plot(min_mass, max_mass, points)
     return retfunc
+
+
+def get_instance_attributes(instance):
+    """Returns a dictionary with all attributes of a given instance.
+
+    Parameters
+    ----------
+    instance : object
+        Instance of a class.
+
+    Returns
+    -------
+    dictionary
+        Dictionary with all attributes of the given instance.
+    """
+    attributes_dict = dict()
+    members = inspect.getmembers(instance)
+    for member in members:
+        # Filter out methods and protected attributes
+        if not inspect.ismethod(member[1]) and not member[0].startswith("__"):
```

Comment:
Just to make sure this is the intended behavior, "weakly private" attributes that start with only one `_` are being added to the dict (the `member[0].startswith("__")` has two underscores).

Reply:
Yes, this is intended. Things like `rocket._csys` is pretty important to have